plugins/hybrid_analysis/plugin.spec.yaml

```diff
@@ -1,145 +1,1109 @@
+      required: false
+    error_origin:
+      title: Error Origin
+      type: string
+      description: Error origin
+      required: false
+    error_type:
+      title: Error Type
+      description: Type of error that occurred
+      type: string
+      required: false
+    job_id:
+      title: Job ID
+      type: string
+      description: Job ID which is generated by server
+      required: false
+    sha256:
+      title: SHA256
+      type: string
+      description: SHA256
+      required: false
+    state:
+      title: State
+      type: string
+      description: State in which the analysis is in
+      required: false
+    verdict:
+      title: Verdict
+      type: string
+      description: Verdict
       required: false
 connection:
   url:
     title: URL
-    description: URL
     type: string
-    default: https://www.hybrid-analysis.com/api
+    description: Hybrid Analysis API server URL
     required: true
-    example: https://www.hybrid-analysis.com/api
+    example: https://www.hybrid-analysis.com
+    default: https://www.hybrid-analysis.com
   api_key:
-    type: credential_secret_key
     title: API Key
-    description: API Key
-    required: true
-    example: 275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f
-  api_token:
     type: credential_secret_key
-    title: API Token
-    description: API token
+    description: API key
     required: true
-    example: 3395856ce81f2b7382dee72602f798b642f14140abababab
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
 actions:
   lookup_hash:
-    title: Lookup Hash
-    description: Lookup By Hash
+    title: Lookup by Hash
+    description: Get summary information for a given hash
     input:
       hash:
+        title: Hash
+        description: Hash to lookup. Must be MD5, SHA1, or SHA256
         type: string
-        description: MD5/SHA1/SHA256 Hash
         required: true
-        example: 275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f
+        example: 44d88612fea8a8f36de82e1278abb02f
     output:
       found:
+        title: Found
         description: True if found
         type: boolean
         required: false
       threatscore:
+        title: Threat Score
         description: Threat Score (max found)
         type: integer
         required: false
       reports:
+        title: Reports
         type: '[]report'
         description: Reports
         required: false
+  lookup_terms:
+    title: Search Database
+    description: Search the Hybrid Analysis database
+    input:
+      filename:
+        title: File Name
+        description: File name
+        type: string
+        required: false
+        example: setup.exe
+      filetype:
+        title: File Type
+        description: File type
+        type: string
+        required: false
+        example: docx
+      country:
+        title: Country
+        description: Country must be specified in the ISO 3166-1 standard
```

Comment:
Can this be a enum or list, so that the user can pick instead of errors.

Reply:
The ISO 3166-1 standard currently comprises 249 countries. Are you sure about that????